crates/containerd-shim-wasmedge/src/instance.rs

```diff
@@ -135,6 +135,7 @@ pub fn prepare_module(
     if stdin.is_some() {
         unsafe {
             STDIN_FD = Some(dup(STDIN_FILENO));
+            #[allow(clippy::unnecessary_unwrap)]
```

Comment:
I would suggest that we remove `unwrap()` instead of allowing unncessary_unwrap policy.